README.md

```diff
@@ -122,6 +122,17 @@ all the necessary files into If you do want to build an MSI installer
 with the WiX toolchain, you can download it here:
 http://wixtoolset.org/releases/
 
+### Errors related to CMake in Visual Studio
+
+Depending on what you do Visual Studio might try to rebuild NativeProxy which
+depends on CMake. You can either install [CMake](https://cmake.org/download/)
+and make sure it's added to your PATH or disable the project in Visual Studios.
+
+Having CMake installed as part of Visual Studios is sufficient to run msbuild
+from the `Developer Command Prompt` but not when building from inside VS,
+unless you've added that to your global PATH. Installing CMake externally allow
```

Comment:
Nits: not "Visual Studios" but "Visual Studio"; "allow" should be "allows".

Reply:
Fixed!